spec/requests/graphql/query/enabled_dynamic_fields_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Retrieving Enabled Dynamic Fields', type: :request do
+  let(:project) { FactoryBot.create(:project) }
+  let(:field_set) { FactoryBot.create(:field_set, project: project) }
+  let(:dynamic_field) { FactoryBot.create(:dynamic_field) }
+  let(:default_value) { "exampleDefaultValue" }
+  let(:digital_object_type) { "item" }
+  let!(:enabled_dynamic_field) do
+    FactoryBot.create(:enabled_dynamic_field, project: project, digital_object_type: digital_object_type,
+      dynamic_field: dynamic_field, required: false, default_value: default_value, field_sets: [field_set])
+  end
+
+  include_examples 'requires user to have correct permissions for graphql request' do
+    let(:request) { graphql query(project.string_key, digital_object_type) }
+  end
+
+  context 'when logged in user is an administrator' do
```

Comment:
It might be more useful to change the "administrator" check to "project administrator" or "user who has project permission" because administrators have access to everything all the time.  But if this is just a generic check that's verifying that "someone who has access" gets the expected the response, then maybe that doesn't matter.  Best case scenario, a lower level user check is more likely to catch a permission bug.

Reply:
probably a good guiding principle to test at the lowest level of access expected to work.

Reply:
lowered privileges used in spec to project controbutor

Reply:
👍